src/scripts/app.ts

```diff
@@ -46,6 +46,17 @@ export default class AdvancedBlanks extends (H5P.Question as { new(): any; }) {
   constructor(config: any, contentId: string, contentData: any = {}) {
     super();
 
+    // Set mandatory default values for editor widgets that create content type instances
+    config = extend({
```

Comment:
I noticed too late that you use H5P.jQuery.extend elsewhere. I try to not have dependencies to jQuery because of [https://h5ptechnology.atlassian.net/browse/HFP-874](https://h5ptechnology.atlassian.net/browse/HFP-874), but if you prefer, I can get rid of that replacement and use H5P.jQuery.extend

Reply:
No, I'm not too keen on using jQuery as well.